examples/mnist-cnn.py

```diff
@@ -53,23 +53,21 @@ def __init__(self, x_test):
 
     def on_train_begin(self, logs=None):
         st.header("Summary")
-        empty_chart = pd.DataFrame({"loss": [], "acc": []})
-        self._summary_chart = st.area_chart(empty_chart)
+        self._summary_chart = st.area_chart()
         self._summary_stats = st.text("%8s :  0" % "epoch")
         st.header("Training Log")
 
     def on_epoch_begin(self, epoch, logs=None):
         self._ts = time.time()
         self._epoch = epoch
         st.subheader("Epoch %s" % epoch)
-        empty_chart = pd.DataFrame({"loss": [], "acc": []})
-        self._epoch_chart = st.line_chart(empty_chart)
+        self._epoch_chart = st.line_chart()
         self._epoch_progress = st.info("No stats yet.")
         self._epoch_summary = st.empty()
 
     def on_batch_end(self, batch, logs=None):
         rows = pd.DataFrame(
-            [[logs["loss"], logs["accuracy"]]], columns=["loss", "accuracy"])
+            [[logs["loss"], logs["acc"]]], columns=["loss", "acc"])
```

Comment:
Change back to "accuracy' and upgrade your Keras to the latest version. They changed this, apparently.

```
pip install keras --upgrade
```